src/lib.rs

```diff
@@ -1921,6 +1921,16 @@ impl Connection {
 
         self.max_rx_data_next = self.max_rx_data_next.saturating_add(read as u64);
 
+        let readable = stream.readable();
+
+        if stream.recv.almost_full() {
+            self.streams.push_almost_full(stream_id);
+        }
+
+        if !readable {
```

Comment:
```suggestion
        if !stream.readable() {
```

This makes it more consistent with similar code around line 2000+

Reply:
See https://github.com/cloudflare/quiche/pull/134#discussion_r318059640 for why this can't be done.